src/Termonad/App.hs

```diff
@@ -379,9 +384,15 @@ forceQuit mvarTMState = do
 setupTermonad :: TMConfig -> Application -> ApplicationWindow -> Gtk.Builder -> IO ()
 setupTermonad tmConfig app win builder = do
   let iconByteString = $(embedFile "img/termonad-lambda.png")
-  iconBytes <- bytesNewTake (Just iconByteString)
-  iconPixelbuf <- pixbufNewFromBytes iconBytes ColorspaceRgb False 8 16 16 (16 * 3)
-  windowSetIcon win (Just iconPixelbuf)
+  let dynamicImage = decodePng iconByteString
+  case dynamicImage of
+    Left msg ->  die "Error happend while getting the icon."
+    Right image -> do
+      let img = convertRGBA8 image
```

Comment:
The RGBA8 format uses 4 bytes per pixel, but the numbers below assume 3 bytes per pixel. You either switch to RGB8 or to change the `3 * 16` to `4 * 16`.

Reply:
Done.

Reply:
Taking a closer look, `img/termonad-lambda.png` is a 256x256 RGBA8 image, so you also need to replace the three `16`s with `256`s.

Reply:
Done.